docs/about/05_archetype.adoc

```diff
@@ -0,0 +1,64 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Helidon Application Bootstrapping
+:description: Helidon Application Bootstrapping
+:keywords: helidon archetypes
+
+== Introduction
+
+<<./05_cli.adoc, Helidon CLI>> provides a convenient way to bootstrap Helidon applications. It allows you to choose from a set of archetypes i.e. application with pre-defined feature sets and lets you customize it by providing host of options.
+
+== Archetypes
+
+Helidon provides following set of archetypes to bootstrap your application development journey.
+
+=== Bare
+It creates a minimal Helidon project, suitable to start from scratch. It includes only required dependencies e.g. in case of Helidon MP, it uses `helidon-microprofile-core` bundle instead of `helidon-microprofile` bundle and adds required dependencies to it. It also doesn't have out-of-box observability setup.
+
+=== QuickStart
+It builds on `Bare` to include multiple REST operations along with default observability setup and a set of additional dependencies to enable ease of development e.g. in case of Helidon MP, it uses `helidon-microprofile` bundle instead of `helidon-microprofile-core` bundle.
+
+=== Database
+It builds on `QuickStart` to demonstrates how to integrate with database (in-memory H2, by default). In case of, Helidon SE that uses the DbClient API while for Helidon MP that uses JPA.
```

Comment:
to demonstrate. (drop the s)

Reply:
Also it seems you have a misplaced period in the middle.

Reply:
Fixed.